docs/cudf/source/user_guide/api_docs/wordpiece_tokenizer.rst

```diff
@@ -0,0 +1,12 @@
+===================
+WordPieceTokenizer
+===================
+.. currentmodule:: cudf.core.wordpiece_tokenizer
```

Comment:
Just noting that in the future, we should make `WordPieceVocabulary` available in the toplevel `cudf` namespace so users don't have to go through `cudf.core` which also contains a lot of private functionality

Reply:
There is a nonzero chance that we'll want to move all non-pandas functionality eventually out of cudf into pylibcudf so that we don't have any of this kind of confusion over namespaces.